### ISSUE TITLE
story(calendar): change week number/day keyboard navigation

### ISSUE THREAD

**Issue by kyubisation** [sbb-design-systems/lyne-components#5253]
In multiple mode the keyboard navigation of the week number/day button navigation should be changed as follows.

First tab into calendar lands onto the top row of week numbers or days and you can navigate to the other numbers/days via arrow keys.
Next tab inside the calendar navigates to the side row of week numbers or days (if available) and you can navigate to the other numbers/days via arrow keys.
Next tab inside the calendar navigates to the days inside the calendar.
